problem/static/problem/js/panel.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function() {
+    $('[data-toggle="tooltip"]').tooltip();
+    $('.progress').css('overflow', 'inherit');
+    $('.del_btn').click(function() {
+        return confirm("Are you sure you want to delete?");
+    });
+    $('.rejudge-btn').click(function() {
+        var submission = $(this).attr('data-submission');
+        return confirm("\
```

Comment:
I suggest to force user to enter number of submissions.
Rejudge the problem only if the user enter correct number.